site/content/contribute/webapp/end-to-end-tests.md

```diff
@@ -1,49 +1,62 @@
 ---
 title: "End-to-End Testing"
-date: 2017-08-20T11:35:32-04:00
+date: 2018-12-04T11:35:32-04:00
 weight: 6
 subsection: Web App
 ---
 
-This page describes how to run End-to-End (E2E) testing and build tests for a section or page of the Mattermost web application.
+This page describes how to run End-to-End (E2E) testing and to build tests for a section or page of the Mattermost web application.  Under the hood, we are using [Cypress](https://www.cypress.io/) which is "fast, easy and reliable testing for anything that runs in a browser."
 
-## Under the Hood
+Not familiar with Cypress? Here are the documentations that will help you get started:
```

Comment:
Could go with `Here is some documentation that` rather than plural `documentations` which I don't think is used as frequently.